### PR TITLE
Add temperature dots sparkline to show relative difference in temperature

### PR DESCRIPTION
**Pull request by RocketPuppy**
2015-07-31-161125_1920x1080_scrot

### CODE REVIEW

File: css/base.css [hunk 1]
```diff
@@ -76,6 +76,17 @@ body table.weather-graph td {
...
+  position: relative;
+}
+
+.hours li .temp-percent {
+  width: 10px;
+  height: 10px;
+  position: absolute;
+  left: 0px;
+  display: block;
+  border-radius: 5px;
```
Comment (samccone) [inline css/base.css]: Beware Old chrome there be prefixes

File: css/base.css [hunk 2]
```diff
@@ -76,6 +76,17 @@ body table.weather-graph td {
...
   width: 12.5%;
   vertical-align: top;
+  position: relative;
+}
+
+.hours li .temp-percent {
+  width: 10px;
+  height: 10px;
+  position: absolute;
+  left: 0px;
```
Comment (samccone) [inline css/base.css]: 0 does not need a unit 🌋🐴

File: js/components/forecast_hours.js [hunk 3]
```diff
@@ -3,13 +3,20 @@ import ForecastHour from "./forecast_hour";
...
 var ForecastHours = React.createClass({
   render: function() {
+    var numberOfHours = 7;
+    var temperatures = this.props.hours.slice(0, numberOfHours + 1).map(function(hour){ return hour.temperature; });
+    var max = Math.max.apply(null, temperatures);
+    var min = Math.min.apply(null, temperatures);
+
     var hours = this.props.hours.map(function (hour, index) {
-      if (index > 7) { return false; }
+      if (index > numberOfHours) { return false; }
```
Comment (samccone) [inline js/components/forecast_hours.js]: I hate that we are abusing the fact that react does not complain with falsy array values. We should really use a reduce here. ⚡️🌚🌇

File: js/components/forecast_hours.js [hunk 4]
```diff
@@ -3,13 +3,20 @@ import ForecastHour from "./forecast_hour";
 
 var ForecastHours = React.createClass({
   render: function() {
+    var numberOfHours = 7;
+    var temperatures = this.props.hours.slice(0, numberOfHours + 1).map(function(hour){ return hour.temperature; });
```
Comment (samccone) [inline js/components/forecast_hours.js]: This line could be so pretty in es6.

**Comment by samccone** [comment]
I defer to @mordecai174 on the design here, the more I look at it the more it looks like where the sun is in the sky :sunrise: :city_sunset:

**Comment by mordecai174** [comment]
@samccone @RocketPuppy - I don't know what design direction we are aiming for. I've done some designs, so if we want to actually deliver on those first, then enhance, or iterate that would be my preference.

Otherwise, free for all, do as you please.

**Comment by joelhawksley** [comment]
@RocketPuppy let's keep this in mind for a future rev once @mordecai174 deems https://github.com/mojotech/helios/issues/33 to be complete. Closing for now.

**Pull request closed without merging**